impl/spanner/ddl.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ktspanner
+
+import (
+	"cloud.google.com/go/spanner/spansql"
+)
+
+//go:generate sh -c "echo '// Code generated by impl/spanner/ddl.go. DO NOT EDIT.' > keytransparency.ddl.go"
+//go:generate sh -c "echo '\npackage ktspanner\n' >> keytransparency.ddl.go"
+//go:generate sh -c "echo 'const ddlString = `' >> keytransparency.ddl.go"
+//go:generate sh -c "cat ./keytransparency.ddl >> keytransparency.ddl.go"
+//go:generate sh -c "echo '`' >> keytransparency.ddl.go"
```

Comment:
Wow, nice hack :)

Would having an explicit shell script which generates the file be more readable though? And here in Go code you would have a single `//go:generate` invoking that script.

Reply:
done